From 316e7316b2d4329eaf91c68389f2e34a452e2f76 Mon Sep 17 00:00:00 2001
From: Eric Lundby <Eric.Lundby@gmail.com>
Date: Mon, 10 Nov 2025 14:25:01 -0700
Subject: [PATCH 1/1] disable-sme-sgemm-kernel

---
 driver/others/dynamic_arm64.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/driver/others/dynamic_arm64.c b/driver/others/dynamic_arm64.c
index 70b51f6fc..d9b297117 100644
--- a/driver/others/dynamic_arm64.c
+++ b/driver/others/dynamic_arm64.c
@@ -472,7 +472,7 @@ static gotoblas_t *get_coretype(void) {
   }
 
 #if !defined(NO_SME)
-  if (support_sme1()) {
+  if (false) {
     return &gotoblas_ARMV9SME;
   }
 #endif
-- 
2.50.1 (Apple Git-155)

